public/class-gdpr-public.php

```diff
@@ -288,8 +288,17 @@ public function update_privacy_preferences() {
 				}
 			}
 		}
-
-		wp_send_json_success();
+		if ( empty ( $_COOKIE ) ) {
+			wp_send_json_error(
+				array(
+					'title'   => esc_html__( 'Warning!', 'gdpr' ),
+					'content' => esc_html__( 'Your browser is set to block cookies therefore we cannot record your acceptance. Please adjust your browser’s settings and try again.', 'gdpr' ),
+				)
+			);
+		} else {
+			wp_send_json_success();
+		}
+	
```

Comment:
We want to move the entire cookie logic to a JS-based approach so caching plugins do not interfere with anything.